snuba/settings/__init__.py

```diff
@@ -261,6 +261,8 @@ class RedisClusters(TypedDict):
 
 # Dataset readiness states supported in this environment
 SUPPORTED_STATES: Set[str] = {"deprecate", "limited", "partial", "complete"}
+READINESS_STATE_MIGRATION_GROUPS_ENABLED: set[str] = set()
```

Comment:
When putting in "temporary" config values I like to leave a comment saying at which condition/date I would expect to remove them in case some big re-prioritization in the company comes through

Reply:
Makes sense. Added!